packages/client/src/intros/shared/Intro.tsx

```diff
@@ -69,7 +73,7 @@ export default ({
         </Paragraph>
       )}
 
-      {exceptions.length > 0 && (
+      {exceptions?.length > 0 && (
```

Comment:
Please remove the question mark. `exceptions` has a default value of `[]` and (if provided) it should be a list of `string[]`.

Reply:
`true`